src/components/TimeSliceGroup.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes, Component } from 'react'
+import SelectableTimeSlice from '../containers/SelectableTimeSlice.jsx'
+import date from '../utils/dates.js'
+
+export default class TimeSliceGroup extends Component {
+  static propTypes = {
+    slices: PropTypes.oneOf([2,3,4,5]),
+    step: PropTypes.number.isRequired,
+    value: PropTypes.instanceOf(Date).isRequired,
+    showlabels: PropTypes.bool,
+    isNow: PropTypes.bool,
+    timeGutterFormat: PropTypes.string,
+    timesliceClassnames: PropTypes.string,
+    culture: PropTypes.string
+  }
+  static defaultProps = {
+    slices: 2,
+    step: 10,
```

Comment:
"visibility" should be a consequence of how tall the sections are which is controlled by User styling correct? I don't think it makes sense to pick hard cap here.

Moreso though, the comment is about he `10` number for step, when the original default was `30` can you talk about why it was changed?